scripts/versions.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
 # Don't export them as they're used in the context of other calls
-avalanche_version=${AVALANCHE_VERSION:-'v1.10.18-rc.0'}
+avalanche_version=${AVALANCHE_VERSION:-'sender-refactor'}
```

Comment:
is this necessary? (only impacts e2e tests)

Reply:
I'm not sure how to get e2e working with a custom tag that isn't an official version tag... haven't dove to see why it doesn't work